spyder/config/user.py

```diff
@@ -674,6 +674,9 @@ def get_defaults_path_name_from_version(self, old_version=None):
             elif check_version(old_version, '52.0.0', '<'):
                 name = '{}-{}'.format(self._defaults_name_prefix, old_version)
                 path = osp.join(get_conf_path(), 'defaults')
+            else:
+                super_class = super(SpyderUserConfig, self)
+                path, name = super_class.get_defaults_path_name_from_version()

```

Comment:
@ccordoba12 that was from my config PR, maybe it was a fix that had to be radded after from the merge?

Reply:
When bumping up the config version (I change the configs for the shortcuts of the Find and Replace so I change it from `51.0.0` to `52.0.0` after the merge) an error raises since the return variables `path` and `name` where not initialized (which I supposed is caused by getting in the `if-else if` statement with a config version with a value more than or equal to `52.0.0`)

Reply:
The thing these lines are nor present in master, that's why I find their addition in this PR a bit odd.

Could you double check if this is needed or not?

Reply:
@dalthviz since I changed to `52.` on that config PR, I think you will need to change to `53.` in that case.

Reply:
Ok, let me double check, but I think @dalthviz just corrected a bug :-p !

Reply:
Lol!! Ok, thanks :+1:

Reply:
@ccordoba12 yes I missed this branch on the else! :-p thanks for fixing the bug @dalthviz ! I will open a PR with a test for this!

Reply:
Although  checking the preferences a warning raise even with this change:

![image](https://user-images.githubusercontent.com/16781833/62175212-5cd0b900-b302-11e9-876d-33aa62365b84.png)

@goanpeca @ccordoba12 probably my fix is not enough. I even tried resetting the preferences of Spyder but the warning keeps appearing

Reply:
We can't merge this PR until that's solved.

@goanpeca, I thought you were going to submit a PR to fix this yesterday. Did you manage to find the cause?

Reply:
@dalthviz can you check this PR to see if it fixes the problems you saw?

https://github.com/spyder-ide/spyder/pull/9936

Reply:
@goanpeca I think it works :+1: